docs/source/reference/authenticators.md

```diff
@@ -37,7 +37,7 @@ with any provider, is also available.
 ## The Dummy Authenticator
 
 When testing, it may be helpful to use the
-:class:`~jupyterhub.auth.DummyAuthenticator`. This allows for any username and
+`jupyterhub.auth.DummyAuthenticator`. This allows for any username and
```

Comment:
I think the myst syntax for this is 

    {class}`~jupyterhub.auth.DummyAuthenticator`

Reply:
Thanks @minrk, I've changed the PR to use this syntax. I wasn't aware of myst. I always assumed the jupyterhub docs were rst & markdown, but #TodayILearned about MyST. Thanks for that too!